spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialFullConvolution.scala

```diff
@@ -252,6 +252,15 @@ class SpatialFullConvolution[T: ClassTag](
     }
   }
 
+  override def computeOutputShape(inputShape: Shape): Shape = {
```

Comment:
Do you use this in keras layer? let's remove it if you have override `computeOutputShape` in keras. The idea is we only touch the code if only necessary for the keras layer.